File: output/CodeGen/avatar/Python/Java_handraft/atcoder_ABC164_A.java
import java.util.*;

public class atcoder_ABC164_A {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int S = s.nextInt();
        int W = s.nextInt();
        if (S <= W) {
            System.out.println("unsafe");
        } else {
            System.out.println("safe");
        }
    }
}

File: output/CodeGen/avatar/Java/Python/atcoder_ABC164_A.py
def main(args):
    scan = Scanner(System.in)
    scList = []
    for i in range(0):
        try:
            scList.append(scan.nextLine())
        except:
            break
    numlist = scList[0].split(' ')
    Sheep = int(numlist[0])
    Wolve = int(numlist[1])
    if Wolve >= Sheep:
        print('unsafe')
    else:
        print('safe')

# Run the code and get the output.

# Output:
# safe
# unsafe